package.json

```diff
@@ -13,8 +13,11 @@
   "scripts": {
     "start": "node index",
     "build": "grunt && handlebars client/views/ -e tpl -f client/js/lounge.templates.js",
-    "test": "HOME=test/fixtures mocha test/**/*.js && npm run lint",
-    "lint": "eslint . && stylelint \"**/*.css\"",
+    "test": "HOME=test/fixtures mocha test/**/*.js",
+    "lint": "npm run lint:js; npm run lint:css",
+    "lint:js": "eslint .",
+    "lint:css": "stylelint \"**/*.css\"",
+    "test-all": "npm run lint; npm run test",
```

Comment:
I'd rather have `test` run everything so that new contributors who are not aware of `test-all` still run the linters:

```js
"test": "mocha test/**/*.js; npm lint:js; npm run lint:css"
```

With a preference to end with the linters because they are silent when they don't trigger an error.